src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -762,7 +762,9 @@ void program::cleanup() {
             }
         }
     }
-    _kernels_cache->reset();
+
+    if (_engine.configuration().kernels_cache_path.empty())
```

Comment:
Why?

Reply:
https://github.com/openvinotoolkit/openvino/pull/13986/files#r1021640422
Reverted. Thank you.